backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/types/CardCreationResultModel.kt

```diff
@@ -0,0 +1,6 @@
+package app.ehrenamtskarte.backend.verification.webservice.schema.types
+
+data class CardCreationResultModel constructor(
+    val dynamicActivationCodeBase64: String?,
+    val staticVerificationCodeBase64: String?,
+)
```

Comment:
This can be discussed again. In my implementation I return `CardCreatioResultModel[]`. But maybe sth like the following makes more sense
```kotlin
data class CardCreationResultModel constructor(
  val dynamicActivationCodesBase64: List<String>
  val staticVerificationCodeBase64: List<String>
)
```